web-ui/src/app/pipes/calendar-date.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DatePipe } from '@angular/common'
+
+@Pipe({
+  name: 'calendarDate'
+})
+export class CalendarDatePipe extends DatePipe implements PipeTransform {
+
+  transform(datetime: number): string {
```

Comment:
Are you sure that we'll always use numbers only? if you see the `DatePipe` support, they allow several types.